src/index.js

```diff
@@ -22,8 +22,9 @@ const trackingId = "UA-164779666-1"; // Replace with your Google Analytics track
 ReactGA.initialize(trackingId);
 ReactGA.pageview(window.location.pathname + window.location.search);
 
+const trackingIdGTM = "GTM-WFVHWFM";
```

Comment:
Where does this value come from? Can we put this in an environment variable? Do we also require any secrets like a token or something/ should we be keeping this value secret?

Reply:
Sorry thats a lot of questions :laughing: